config/database.yml

```diff
@@ -81,6 +81,6 @@ test:
 #
 production:
   <<: *default
-  database: employee_recognition_production
-  username: employee_recognition
-  password: <%= ENV['DATABASE_PASSWORD'] %>
+  database: db_prod
+  username: valar
```

Comment:
to jest wielkie nono - nie powinienes ruszac tego pliku a szcegolnie podawac hasla do proda